rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy

```diff
@@ -157,14 +158,14 @@ class BakePoller implements ApplicationListener<ContextRefreshedEvent> {
 
   void updateBakeStatusAndLogs(String statusId) {
     BakeStatus bakeStatus = executor.updateJob(statusId)
-    Id bakesId
+    def tags = [:]
 
     if (bakeStatus) {
       if (bakeStatus.state == BakeStatus.State.COMPLETED) {
         completeBake(statusId, bakeStatus.logsContent)
-        bakesId = registry.createId('bakes').withTag("success", "true")
+        tags = ["success":"true"]
       } else if (bakeStatus.state == BakeStatus.State.CANCELED) {
-        bakesId = registry.createId('bakes').withTag("success", "false").withTag("cause", "jobFailed")
+        tags = ["success":"false", "cause":"jobFailed"]
```

Comment:
Could  assign each value individually. Not sure why we need to initialize `tags` up above if we're just overwriting it in each branch.